endtoend/endtoend_test.go

```diff
@@ -71,7 +71,8 @@ func runEndToEndTest(t *testing.T, config *types.E2EConfig) {
 	}
 	// Small offset so evaluators perform in the middle of an epoch.
 	epochSeconds := params.BeaconConfig().SecondsPerSlot * params.BeaconConfig().SlotsPerEpoch
-	genesisTime := time.Unix(genesis.GenesisTime.Seconds+int64(epochSeconds/2), 0)
+	// Adding a half slot here to ensure the requests are in the middle of an epoch.
+	genesisTime := time.Unix(genesis.GenesisTime.Seconds+int64(epochSeconds/2+(params.BeaconConfig().SecondsPerSlot/2)), 0)
```

Comment:
this is genesis time + half_epoch + half_slot . Can you clarify why you are doing this , this way ?

Reply:
Sure thing

Reply:
Added clearer comments.